index.html

```diff
@@ -1,253 +1,229 @@
-          <i class="material-icons">menu</i>
-        </a>
-      </nav>
-    </header>
-    <section class="headline">
-      <div class="info-container">
-        <h1 class="title-text">Adams <br />Bamfo</h1>
-        <div>
-          <p class="descriptive-text">
-            Hello! I am a software developer! I can help you build a product,
-            feature or website. Take a look at my work.
-          </p>
-          <p class="descriptive-text">
-            If you like what you see and have a project you need coded, don’t
-            hesitate to contact me.
-          </p>
-        </div>
-        <button class="primary-button">start collaboration</button>
-      </div>
-
-      <img
-        id="mobile-image-section-one"
-        src="/images/section-one-mobile-img.jpg"
-        alt="headline-image"
-      />
-    </section>
-    <section class="work-section">
-      <h1>Projects</h1>
-      <div class="project-container">
-        <img src="./images/project-one.jpg" alt="project-one-image" />
-        <div class="project-bottom-section">
-          <p>Project name goes here</p>
-          <div class="languages-container">
-            <div>HTML/CSS</div>
-            <div id="ruby-text">Ruby on Rails</div>
-            <div>JavaScript</div>
-          </div>
-          <button>
-            <p>see this project</p>
-            <img
-              src="./images/select-project.png"
-              alt="see-project-icon"
-              srcset=""
-            />
-          </button>
-        </div>
-      </div>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/css/homepage-styles.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+ <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Sharp:opsz,wght,FILL,GRAD@48,400,0,0" />
+ 
+  <title>homepage</title>
+</head>
 
-      <div class="project-container">
-        <img src="./images/project-two.jpg" alt="project-one-image" />
-        <div class="project-bottom-section">
-          <p>Project name goes here</p>
-          <div class="languages-container">
-            <div>HTML/CSS</div>
-            <div id="ruby-text">Ruby on Rails</div>
-            <div>JavaScript</div>
-          </div>
-          <button>
-            <p>see this project</p>
-            <img
-              src="./images/select-project.png"
-              alt="see-project-icon"
-              srcset=""
-            />
-          </button>
-        </div>
+<body>
+  <header class="header-container">
+    <img src="/images/image_geometry_1.png" alt="geometry image" />
+    <nav>
+      <a href="#">
+        <i class="material-icons">menu</i>
+      </a>
```

Comment:
- [ ] The hamburger menu is displayed on the left side while it should be on the right, kindly change it to match the requirements.

Reply:
Hi @algerina, I am so grateful for your review. You have done a great review. Please with the hamburger menu review, I am a little confused. I don't know the kind of error it is. I see it on the right when I check from my end.  Please what do you think is the possible problem? 

This is my HTML for that section
![Screenshot_20230220_050306](https://user-images.githubusercontent.com/63151299/220167433-9d9861af-202e-4a1b-911b-3e9727fb90f9.png)

This is the CSS
![Screenshot_20230220_051420](https://user-images.githubusercontent.com/63151299/220167837-e89e3fdb-22c3-47e3-acf7-22b695be8101.png)


![Screenshot_20230220_044848](https://user-images.githubusercontent.com/63151299/220163520-0c647a5e-4436-4706-a079-4688a674137e.png)

Reply:
- [ ] The issue is in the CSS link in your HTML file line 8.